### PR TITLE
TW-948/TW-947: rewording

### PR DESCRIPTION
**Pull request by imGok**
#948 
#947 

image

image

### CODE REVIEW

**Review by drminh2807 — APPROVED** [review]

**Comment by nqhhdev** [comment]
Pls add demo, thanks

**Comment by imGok** [comment]
> Pls add demo, thanks

done

File: lib/pages/chat/chat_context_menu_actions.dart [hunk 1]
```diff
@@ -19,7 +19,7 @@ enum ChatContextMenuActions {
         return L10n.of(context)!.copyMessageText;
       case ChatContextMenuActions.pinMessage:
```
Comment (hoangdat) [inline lib/pages/chat/chat_context_menu_actions.dart:20]: please change the name of the action also

File: lib/pages/chat/chat_context_menu_actions.dart [hunk 2]
```diff
@@ -19,7 +19,7 @@ enum ChatContextMenuActions {
...
         return unpin
-            ? L10n.of(context)!.unpinThisMessage
+            ? L10n.of(context)!.unpinThisChat
             : L10n.of(context)!.pinMessage;
```
Comment (hoangdat) [inline lib/pages/chat/chat_context_menu_actions.dart:23]:
```suggestion
            : L10n.of(context)!.pinChat;
```

**Review by sherlockvn — APPROVED** [review]

**Review by nqhhdev — APPROVED** [review]

**Review by hoangdat — APPROVED** [review]

**Pull request merged**